jobs/validate-kubeflow-microk8s/Jenkinsfile

```diff
@@ -55,24 +55,26 @@ pipeline {
                 exec "sudo snap install lxd"
                 exec "sudo snap install microk8s --classic --channel ${params.microk8s_channel}"
                 exec "sudo snap install yq"
-                exec "sudo apt update && sudo apt install -y python-tox"
+                exec "sudo apt update && sudo apt install -y python-tox docker.io"
             }
         }
         stage('Deploy kubeflow') {
             steps {
-                exec "microk8s.status --wait-ready"
+                exec "microk8s.status --wait-ready --timeout 300"
                 exec "microk8s.enable dns storage registry"
+                exec 'microk8s.kubectl patch configmap -n kube-system kube-dns -p "$(<jobs/validate-kubeflow-microk8s/nameservers.json)"'
```

Comment:
I do not see the mtu configuration

Reply:
I left off the MTU configuration to test if it's actually necessary. Unfortunately I can't test that without this PR.

Reply:
@knkski I think that if you go to the configure option in jenkins you can point your job to use a branch. This helps accelerate the dev of jenkins jobs. @battlemidget, we can still do that, right?